test/test_helper.rb

```diff
@@ -3,7 +3,9 @@
 require 'rails/test_help'
 require 'capybara/rails'
 require 'mocha/mini_test'
+require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.